lib/widgets/file_widget/download_file_tile_widget.dart

```diff
@@ -63,11 +65,13 @@ class DownloadFileTileWidget extends StatelessWidget {
                     width: style.iconSize,
                     height: style.iconSize,
                     decoration: BoxDecoration(
-                      color: Theme.of(context).colorScheme.primary,
+                      color: hasError
+                          ? Theme.of(context).colorScheme.error
+                          : Theme.of(context).colorScheme.primary,
                       shape: BoxShape.circle,
```

Comment:
i think you should put it in `MessageFileTileStyle`

Reply:
done